apps/govern/components/VeOlas/MaxButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button, Typography } from 'antd';
+
+import { getCommaSeparatedNumber } from 'common-util/functions';
+
+const { Text } = Typography;
+
+type MaxButtonProps = { olasBalance?: string; onMaxClick: () => void };
+
+export const MaxButton = ({ olasBalance, onMaxClick }: MaxButtonProps) => {
+  return (
+    <Text type="secondary">
+      Balance:&nbsp;
+      {getCommaSeparatedNumber(olasBalance || 0)}&nbsp;OLAS&nbsp;
```

Comment:
feels off to show a 0 value for balance, as it will be inaccurate if the user has funds.
something like a `-` or `--` or m-dash, may be more appropriate for letting user know the balance is yet to load

```suggestion
      {getCommaSeparatedNumber(olasBalance || --)}&nbsp;OLAS&nbsp;
```